Formula/xcbuild.rb

```diff
@@ -0,0 +1,22 @@
+class Xcbuild < Formula
+  desc "Xcode-compatible build tool"
+  homepage "https://github.com/facebook/xcbuild"
+  url "https://github.com/facebook/xcbuild.git", :tag => "0.1.0",
```

Comment:
Please split each `:` onto its own line. Having everything indented over to the far right is both pretty grim looking & blasts past our desired ~80 character width.